pkg/bootkube/bootkube.go

```diff
@@ -94,6 +95,8 @@ func NewBootkube(config Config) (*bootkube, error) {
 }
 
 func (b *bootkube) Run() error {
+	fmt.Println("Running temporary bootstrap control plane...")
```

Comment:
nit: Can we have a wrapper func like `stdout()` to be explicit about our intention for these vs loglines?